core/src/test/java/org/elasticsearch/index/query/HasChildQueryParserTests.java

```diff
@@ -26,23 +26,23 @@
 
 public class HasChildQueryParserTests extends ESTestCase {
     public void testMinFromString() {
-        assertThat("fromString(min) != MIN", ScoreMode.Min, equalTo(HasChildQueryParser.parseScoreMode("min")));
+        assertThat("resolve(min) != MIN", ScoreMode.Min, equalTo(HasChildQueryParser.parseScoreMode("min")));
```

Comment:
Why these changes? I first thought it's because the method rename but it seems unrelated to your change now. If so, can you change this back?